main/main.cpp

```diff
@@ -189,6 +189,7 @@ static bool dump_extension_api = false;
 #endif
 bool profile_gpu = false;
 
+Color boot_bg_color;
```

Comment:
Any reason for this to be a global variable? It probably should be in the `setup2`.

Reply:
You're right, I'll fix it later